src/ruby/yast/y2logger.rb

```diff
@@ -40,6 +75,37 @@ def initialize(*_args)
       # process also debug messages but might not be logged in the end
       self.level = ::Logger::DEBUG
     end
+
+    # log a block of commands, adds a special begin and end markers into the log,
+    # the block should be one big logical step in the process,
+    # can be used recursively, e.g. log.group might call another log.group inside
+    # @param description [String] short description of the block
+    # @param block [Proc] block to call
+    # @yield param group [LogGroupResult] can be optionally used to pass result details
+    def group(description, &block)
+      details = LogGroupResult.new
+      # mark start of the group
+      info "::group::#{Process.clock_gettime(Process::CLOCK_MONOTONIC)}::#{description}"
+
+      if block_given?
+        ret = block.call(details)
+      else
+        raise ArgumentError, "Missing block parameter"
```

Comment:
"block parameter" is ambiguous, without context I could interpret it as a parameter **of** a block

```suggestion
        raise ArgumentError, "Missing a block"
```